libr/bin/obj.c

```diff
@@ -371,13 +374,13 @@ R_API int r_bin_object_delete(RBin *bin, ut32 binfile_id, ut32 binobj_id) {
 }
 
 R_API void r_bin_object_set_baddr(RBinObject *o, ut64 baddr) {
-	if (!o || baddr == UT64_MAX) {
-		return;
-	}
+	r_return_if_fail (o && baddr != UT64_MAX);
 	o->baddr_shift = baddr - o->baddr;
```

Comment:
XXX: this is not setting `o->baddr` anymore. It should, given the name of the API. All changes to baddr should come through this API. Also, `baddr` is many time initialized with `0xfffffffffffffffffff`, but why? A baddr is always defined, even it is not set explicitly (where, I think, it is assumed to be zero. So dealing with 0xffffff..fff just makes things harder)